en/docs/learn/enterprise-integration-patterns/message-construction/format-indicator.md

```diff
@@ -1,10 +1,10 @@
 # Format Indicator
```

Comment:
Shall we follow a similar format as https://github.com/wso2/docs-mi/pull/743/files here as well?

Reply:
Updated